sirepo/job_supervisor.py

```diff
@@ -168,16 +168,16 @@ def init_module(**imports):
                 "maximum run-time for sequential job",
             ),
         ),
-        purge_non_premium_after_secs=(
-            0,
-            pkconfig.parse_seconds,
-            "expiration period for purging non-premium users simulations",
-        ),
-        purge_non_premium_task_secs=(
+        purge_check_interval=(
             None,
             pkconfig.parse_seconds,
             "time interval to clean up simulation runs of non-premium users (%H:%M:%S)",
```

Comment:
`(%H:%M:%S)` is implicit from `pkconfig.parse_seconds`. What we need to do is have `pkconfig` output a help file from the descriptions, names, defaults, and types (parse_seconds).

I would add that `0` means no checks are performed.